src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,62 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.6.0
+13 March, 2023
+
+This version comes with a breaking change. The default ingress class for ingresses managed by okteto is `okteto-nginx` instead of `nginx` to prevent colissions with other controllers that might be already installed in the cluster. All ingresses managed by Okteto will be automatically updated. No action is required from the user. If you want to keep the previous value (`nginx`), you would need to set the following values in your helm values file:
```

Comment:
```suggestion
This version comes with a breaking change. The default ingress class for ingresses managed by okteto changed from `nginx` to `okteto-nginx`  to prevent collisions with other controllers that might be already installed in the cluster. There is no action required from the user,  all ingresses managed by Okteto will be automatically updated as part of the upgrade.

If you want to continue to use the previous value (`nginx`), you can do it by setting the following values in your helm values file:
```